java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/launchers/JUnitLaunchConfigurationDelegate.java

```diff
@@ -52,6 +56,23 @@ public JUnitLaunchArguments getJUnitLaunchArguments(ILaunchConfiguration configu
         }
     }
 
+    private String[] getVmArguments(VMRunnerConfiguration config) {
+        final List<String> vmArgs = new ArrayList<>();
+        vmArgs.addAll(Arrays.asList(config.getVMArguments()));
+        
+        if (config.isPreviewEnabled()) {
+            vmArgs.add("--enable-preview");
+        }
+
+        final String dependencies = config.getOverrideDependencies();
```

Comment:
how is it correlated with module path support?

Reply:
The logic is referred from jdt. See:

https://github.com/eclipse/eclipse.jdt.ui/blob/afb285d86adfc259b684d907eb153a874448ccbc/org.eclipse.jdt.junit.core/src/org/eclipse/jdt/junit/launcher/JUnitLaunchConfigurationDelegate.java#L223-L230

https://github.com/eclipse/eclipse.jdt.debug/blob/205c5a0c1fcd1d7b72a0af85c79632f7b522d90a/org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/StandardVMDebugger.java#L234-L241